### DISCUSSION TITLE
Reverse Tab Order from Bottom to Top when using Bottom Toolbar

### DISCUSSION THREAD

**Discussion by Raul1198** [bromite/bromite#2327] [Ideas]
When using Bottom Toolbar it would be easier for one hand  reachability when the tabs are arranged from bottom to top like 
![Screenshot_20220916-001702_Bromite](https://user-images.githubusercontent.com/29801315/190487684-d15d46a1-32e9-4c73-b2fb-9bfb252d4cba.png)
 Instead of the default one 
![Screenshot_20220916-001807_Bromite](https://user-images.githubusercontent.com/29801315/190487796-cd9ab851-8a94-426e-a840-bd96fe0c5704.png)

**Comment by uazo** [comment]
some time ago I tried to develop it but then I saw that it is quite complex.
however it remains in the things that sooner or later I will cover.

**Reply by Raul1198** [reply]
thanks, really appreciated

**Comment by levma** [comment]
Please make a config option for it. It seems that this is now the default behavior and i don't know how to change it. Maybe it works good for you but for me it is a worse UX:
1. I'm used to read/write from left to right and from top to bottom. This order is the opposite of that and it's hard for me to wrap my mind around it. It just feels wrong.
2. It's way harder to select the current tab(s) with my thumb as they moved to the top of the screen. Why did we all wanted to have the bottom bar in the first place? Of cpurse because its way easier to access with the thumb. This enhanced accessibility seems to be undone by the last UX change...

**Comment by csagan5** [comment]
Please do not use discussions for feature requests and bug reports, see https://github.com/bromite/bromite/discussions/893
